test/logger.test.js

```diff
@@ -32,14 +30,14 @@ describe('strigo-node-logger', () => {
     });
 
     it('should not write debug printout when set up with info level', () => {
-      const log = setupNodeLogger({ env: 'prod', level: 'info' });
+      const log = setupNodeLogger({});
```

Comment:
You should set up `level` explicitly here. It keeps the test sane if we ever change the default level, and makes it clear it's intentional.